source/javascripts/directives/image_loader.coffee

```diff
@@ -0,0 +1,15 @@
+EGA.directive 'imageSuggestions', ->
```

Comment:
I am not quite happy with the scope of the directive. In my opinion this one should actually be its own element by setting `restrict: 'E'` because it should work just like a normal image tag, except that it selects the first working image from an array. Also the name could be improved i think … maybe `<sanitized-image src="award.images" />` or even something future-thinking like `<slideshow src="award.images" />` because you could also see it as a slideshow that doesn't have full functionality yet :-D